addons/kube-ingress-aws-controller/README.md

```diff
@@ -87,13 +87,14 @@ kube-ingress-aws-controller, which we will use:
   "Effect": "Allow",
   "Action": [
     "acm:ListCertificates",
+    "acm:GetCertificate",
```

Comment:
I checked; this is only the certificate (not the key), so no security concerns IMO.  :+1: